### PR TITLE
Mar 3468

### PR DESCRIPTION
**Pull request by Reshma-M-A**
This pull request is adding a Teams screen in the admin page. 

The team screen shows the: 
- Who's in the team
- Breakdown of team leads / team members
- Who has permission to be creating drafts, publishing, downloading responses etc.

### CODE REVIEW

File: app/templates/index.html [hunk 1]
```diff
@@ -67,6 +67,16 @@
                   <input type="submit" value="Search" class="button-save">
               </form>
 
+              <form action="{{ url_for('.find_team_by_team_id') }}" method="get" class="question">
+                <label class="question-heading" for="team_id">Find Teams’ ID</label>
+                <p class='hint'>
+                  You can find this number at the end of the Teams’ URL.
```
Comment (tobiasbrown) [inline app/templates/index.html]: Will users know what this URL is?  Should we provide an example?

File: app/templates/view_teams.html [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+<div>
+  <h3>Team Details</h3>
+  Team ID: {{team_id}}
+  <br/>
+  Team Name: {{team.name}}
+</div>
+
+<div>
+  {% for i in team.teamLeads %}
+  <h3> Team Leads Information</h3>
```
Comment (tobiasbrown) [inline app/templates/view_teams.html]:
```suggestion
  <h3>Team Leads Information</h3>
```

File: app/templates/view_teams.html [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    {% for key, value in team.teamLeads[i].items() %}
```
Comment (tobiasbrown) [inline app/templates/view_teams.html]: Check the indentation for this `for` block.  It looks a bit off to me.  Applies to another blocks below.

File: app/templates/view_teams.html [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+        {% else %}
+          name : {{ value }}
+          <br/>
+          {% endif %} 
+      {% endfor %}
+  {% endfor %}
+</div>
+
+<div>
+  <h3> Team Members</h3>
```
Comment (tobiasbrown) [inline app/templates/view_teams.html]:
```suggestion
  <h3>Team Members</h3>
```

File: app/templates/view_teams.html [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+  {% for i in team.teamMembers %}
+      {% for key, value in team.teamMembers[i].items() %}
+          {% if key == 'emailAddress'%}
+            email address: {{value}}
+            <br/>
+         {% elif key == 'name'%}
+            name : {{value}}
+            <br/>
+            <br/>
+          {% endif %}
```
Comment (tobiasbrown) [inline app/templates/view_teams.html]: Would it be easier to remove the `if` block and do something like this?
```html
{% for item in team.teamMembers[i].items() %}
    email address: {{ item['emailAddress'] }}
    <br/>
    name: {{ item['name'] }}
    <br/>
{% endfor %}

File: app/templates/view_teams.html [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+      {% endfor %}
+    {% endfor %}
+</div>
+
+<div> 
+    <h3> Team Members' Permissions</h3>
```
Comment (tobiasbrown) [inline app/templates/view_teams.html]:
```suggestion
    <h3>Team Members' Permissions</h3>
```

File: app/templates/view_teams.html [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+                {{value}}
+                <br/>
+                <br/>
+            {% endif %}
+      {% endfor %}
+    {% endfor %}
+</div>
+
+<div>
+  <h3> Brief: </h3>
```
Comment (tobiasbrown) [inline app/templates/view_teams.html]:
```suggestion
  <h3>Brief:</h3>
```

File: app/templates/view_buyers.html [hunk 8]
```diff
@@ -207,7 +209,7 @@ <h3> Edit brief data</h3>
       <br/>
       <input type="submit" value="Search" class="button-save">
   </form>
-  <br />
+  <br />
```
Comment (tobiasbrown) [inline app/templates/view_buyers.html]:
```suggestion
  <br />
```

File: app/main/views/buyers.py [hunk 9]
```diff
@@ -250,3 +250,18 @@ def seller_feedback_email(brief_id):
...
+@main.route('/team', methods=['GET'])
+@login_required
+@role_required('admin')
+def find_team_by_team_id():
+    team_id = request.args.get('team_id')
+
+    team = data_api_client.req.admin().team(team_id).get()
+
+    return render_template_with_csrf(
+        "view_teams.html",
```
Comment (tobiasbrown) [inline app/main/views/buyers.py]: Should this template be named `view_team` if we're displaying data for a single team?

Comment (Reshma-M-A) [reply app/main/views/buyers.py]: Made these changes.

**Pull request closed without merging**